src/RpcServer/RpcServer.cs

```diff
@@ -119,7 +119,8 @@ public void StartRpcServer()
             .Configure(app =>
             {
                 app.UseResponseCompression();
-                app.Run(ProcessAsync);
+                app.Use(ProcessAsync);
+                app.Run(InvalidRequestAsync);
```

Comment:
I think we should use `InvalidRequestAsync` first and then run `ProcessAsync` as the terminal middleware. Then `InvalidRequestAsync` can catch the exceptions in `ProcessAsync`.

Reply:
If we're going to make `ProcessAsync` the terminal middleware, then I think we should simply go back to [the original commit in this PR](https://github.com/neo-project/neo-modules/pull/687/commits/bb4cccad4c49502ed5f776d1b5493e9fa956f2b9) and make `ProcessAsync` public. 

I split `InvalidRequestAsync` out from `ProcessAsync` so Express could insert its middleware in the ASP.NET Pipeline after `ProcessAsync`. If `ProcessAsync` is terminal middleware, then Express has no choice but to put its middleware before `ProcessAsync`. Compatibility with `RpcServer` is a high priority for Express, so I wanted to give `ProcessAsync` the highest priority position in the ASP.NET pipeline. But that doesn't work if `ProcessAsync` can only be used as terminal middleware.

Honestly, I don't care which approach we take. Simply making `ProcessAsync` public is smaller and has no logic changes. Changing `ProcessAsync` to be non-terminal means for less code for Express. Either approach works for me.